src/components/Inquiry/Answer.jsx

```diff
@@ -1,60 +1,69 @@
-import React, { useState } from 'react'
-import { useSelector } from 'react-redux'
-import CustomIconButton from '../custom/CustomIconButton'
-import store from '../../state/store/configureStore'
-import CustomRadioButton from '../../components/custom/CustomRadioButton'
-
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { animateScroll } from 'react-scroll';
+import CustomIconButton from '../custom/CustomIconButton';
+import store from '../../state/store/configureStore';
+import CustomRadioButton from '../../components/custom/CustomRadioButton';
+import CustomSelectInput from '../custom/CustomSelectInput';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
 const Answer = ({
   text,
   type,
   placeholder,
   questionKey,
   labels,
   dataCys,
-  values, 
+  values,
 }) => {
   const [inputValue, setInputValue] = useState(
     store.getState().formData[questionKey]
-  )
-  const filled = useSelector((state) => state.filledAnswers[questionKey])
+  );
+  const filled = useSelector((state) => state.filledAnswers[questionKey]);
+  const isSmall = useMediaQuery('(max-width:620px)');
   const setAnswer = (event) => {
-    event.preventDefault()
+    event.preventDefault();
     if (filled) {
       store.dispatch({
         type: 'SWITCH_ANSWER_FILLED_STATUS',
         payload: { key: questionKey, filled: false },
-      })
+      });
     } else {
+      animateScroll.scrollMore(isSmall ? 280 : 200);
```

Comment:
are all containers the same size for scroll? can we take the height of the container and put it into variable?

Reply:
I mean we could, but might be over complicating it for 8 questions.